### PR TITLE
Add Foreman 2.0 config

### PR DESCRIPTION
**Pull request by zjhuntin**

### CODE REVIEW

File: configs/foreman/2.0.yaml [hunk 1]
```diff
@@ -0,0 +1,535 @@
...
+  - foreman-client-2.0-rhel5
+  - foreman-client-2.0-fedora29
+  - foreman-client-2.0-sles11
+  - foreman-client-2.0-sles12
+:repos: []
+:gpg_key:
+:strict_keys: true
+:tags:
+  - name: foreman-rails-2.0-rhel7
+    based_off: tfm-ror52-rhel7
```
Comment (ehelms) [inline configs/foreman/2.0.yaml]: We shouldn't need the `-rails` tags anymore.

File: configs/foreman/2.0.yaml [hunk 2]
```diff
@@ -0,0 +1,535 @@
+---
+:project: foreman
+:github_org: theforeman
+:releases:
+  :2.0.0:
+    :redmine_version_id: 1132
+:mash_scripts:
+  - foreman-2.0-rhel7-dist
+  - foreman-plugins-2.0-rhel7-dist
+  - foreman-rails-2.0-rhel7
```
Comment (ehelms) [inline configs/foreman/2.0.yaml]: Can drop this mash script with the work you did

File: configs/foreman/2.0.yaml [hunk 3]
```diff
@@ -0,0 +1,535 @@
...
+        - patch
+        - redhat-rpm-config
+        - rh-ruby25-scldevel
+        - rpm-build
+        - scl-utils-build
+        - sed
+        - shadow-utils
+        - tar
+        - tfm-build
+        - tfm-ror52-scldevel
```
Comment (ehelms) [inline configs/foreman/2.0.yaml]: This can be dropped.

File: configs/foreman/2.0.yaml [hunk 4]
```diff
@@ -0,0 +1,535 @@
...
+        - centos-release
+        - foreman-build
+        - redhat-rpm-config
+        - rh-ruby25-scldevel
+        - rhpkg-simple
+        - rpm-build
+        - scl-utils-build
+        - shadow-utils
+        - tfm-build
+        - tfm-ror52-scldevel
```
Comment (ehelms) [inline configs/foreman/2.0.yaml]: Similarly dropped

**Comment by zjhuntin** [comment]
@ehelms updated! 

@ekohl Updated in this PR https://github.com/theforeman/tool_belt/pull/240

**Comment by tbrisker** [comment]
travis is green, probably the updating hook failed.
Thanks @zjhuntin !

**Pull request merged**